api/auth/user_management.py

```
@@ -222,22 +147,49 @@ async def update_identity_last_login(provider, provider_user_id):
+    # Check Authorization header
+    auth_header = (
+        request.headers.get("authorization")
+        or request.headers.get("Authorization")
+    )
+    if auth_header:
+        try:
+            parts = auth_header.split(None, 1)
+            if len(parts) == 2 and parts[0].lower() == "bearer":
+                return parts[1].strip()
+        except Exception:  # pylint: disable=broad-exception-caught
+            pass
```

Ruff (0.12.2)

`try`-`except`-`pass` detected, consider logging the exception

(S110)